src/webinars/2023/industrial-data-node-red.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Connect, Integrate, Visual Industrial Production Metrics with Node-RED"
```

Comment:
```suggestion
title: "Connect, Integrate & Visualize Industrial Production Metrics with Node-RED"
```

Sentence structure didnt feel right (though I couldn't put my finger on why)

Open to debate.